hydromt_wflow/workflows/demand.py

```diff
@@ -374,7 +374,7 @@ def allocation_areas(
         ds_like["basins"] != ds_like["basins"].raster.nodata,
         allocation_areas.raster.nodata,
     )
-    allocation_areas.name = "allocation_areas"
+    allocation_areas.name = "demand_allocation_area_id"
```

Comment:
Undo. The renaming from hydromt convention to staticmap name should happen in wflow.py normally